solution/DecReprSeniorCoding.rb

```diff
@@ -0,0 +1,10 @@
+class DRSC
+    def initialize(number)
```

Comment:
Please refer to https://github.com/rubocop/ruby-style-guide for coding style kub, for example, we use 2 tabs instead of 4, single quota instead of double quotes.

You can also use `rubocop` gem for auto-formatting the codes as well
I guess it just simply install `gem install rubucop` and then `rubocop -a file_path`